data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/paths/_teku_v1_admin_add_peer.json

```diff
@@ -0,0 +1,46 @@
+{
+  "post" : {
+    "tags" : [ "Teku" ],
+    "operationId" : "AddPeer",
+    "summary" : "Add a static peer to the node",
+    "description" : "Add a static peer to the node.",
+    "requestBody" : {
+      "content" : {
+        "application/json" : {
+          "schema" : {
+            "type" : "array",
```

Comment:
we'll just want a single entry here now, still showing an array

Reply:
Fixed